pkg/repositories/gormimpl/resource_repo.go

```diff
@@ -20,6 +20,8 @@ type ResourceRepo struct {
 	metrics          gormMetrics
 }
 
+const priorityDescending = "priority desc"
```

Comment:
nit: Is there an `ID.Priority` that has the `priority` string to use here?

Reply:
from the docs it looks like you still have to explicitly use the string field name :\

on the plus side, model fields should change infrequently :-)